docs/network/migration-guide.md

```diff
@@ -119,12 +127,70 @@ class LitV3 {  // or Class LitV4
     this.litNodeClient = client;
   }
 
+  async getSessionSignatures(){
+      // Connect to the wallet
+      const ethWallet = new ethers.Wallet(
+        "<your private key>"
+      );
```

Comment:
same as above

Reply:
[Moved it](https://github.com/LIT-Protocol/docs/pull/279/commits/4f4aa2b5c0121389d4e8efabdab4791ae28ffd78) into the `constructor`